color_refinement.py

```diff
@@ -148,21 +148,26 @@ def fast_color_refine(graph: Graph, coloring: Coloring) -> Coloring:
                 split_count += 1
                 # Each color_class is added to the list
                 new_color_classes.append(new_color)
-            # The smallest_color here is the first color added to the list, which is the original color
-            smallest_color = new_color_classes[0]
+            # The largest_color here is the first color added to the list, which is the original color
```

Comment:
Vervang met: "Initialize 'largest_color' met de eerste color class"
Want nu suggesteerd het dat de eerste altijd de grootste is.